src/azure-cli/azure/cli/command_modules/redis/tests/latest/test_redis_scenario.py

```diff
@@ -108,6 +109,114 @@ def test_redis_cache_with_redisversion(self, resource_group):
+        self.assertTrue(result['accessPolicyName'] == self.kwargs['access-policy-name'])
+        self.assertTrue(result['objectId'] == self.kwargs['object-id'])
+        self.assertTrue(result['objectIdAlias'] == self.kwargs['object-id-alias1'])
+
+        # List access policy assignments
+        result = self.cmd('az redis access-policy-assignment list -n {name} -g {rg}').get_output_in_json()
+        self.assertTrue(len(result) == 1)
+
+        # Update access policy assignment
+        result = self.cmd('az redis access-policy-assignment update -n {name} -g {rg} --assignment-name {access-policy-assignment-name} --access-policy-name {access-policy-name} --object-id {object-id} --object-id-alias {object-id-alias2}').get_output_in_json()
+        self.assertTrue(result['name'] == self.kwargs['access-policy-assignment-name'])
+        self.assertTrue(result['accessPolicyName'] == self.kwargs['access-policy-name'])
+        self.assertTrue(result['objectId'] == self.kwargs['object-id'])
+        self.assertTrue(result['objectIdAlias'] == self.kwargs['object-id-alias2'])
+
+        # Get access policy assignment
+        result = self.cmd('az redis access-policy-assignment show -n {name} -g {rg} --assignment-name {access-policy-assignment-name}').get_output_in_json()
+        self.assertTrue(result['name'] == self.kwargs['access-policy-assignment-name'])
+        self.assertTrue(result['accessPolicyName'] == self.kwargs['access-policy-name'])
+        self.assertTrue(result['objectId'] == self.kwargs['object-id'])
+        self.assertTrue(result['objectIdAlias'] == self.kwargs['object-id-alias2'])
+
+        # Delete access policy assignment
+        self.cmd('az redis access-policy-assignment delete -n {name} -g {rg} --assignment-name {access-policy-assignment-name}')
+
+        # List access policy assignments
+        result = self.cmd('az redis access-policy-assignment list -n {name} -g {rg}').get_output_in_json()
+        self.assertTrue(len(result) == 0)
+
+        # Delete access policy
+        self.cmd('az redis access-policy delete -n {name} -g {rg} --access-policy-name {access-policy-name}')
+
+        # List access polices
+        result = self.cmd('az redis access-policy list -n {name} -g {rg}').get_output_in_json()
+        self.assertTrue(len(result) == 3)
+
+        # Commenting out due to issues with tearing down test for update (need to provide exact sleep time for lro to complete)
+        """
+        # Disable aad on cache
+        self.cmd('az redis update -n {name} -g {rg} --set redisConfiguration.aadEnabled=false --no-wait False')
+        result = self.cmd('az redis show -n {name} -g {rg}').get_output_in_json()
+
+        # Verify cache is aad disabled
+        print(result)
+        if self.is_live:
+            while result['provisioningState'] == 'ConfiguringAAD':
+                result = self.cmd('az redis show -n {name} -g {rg}').get_output_in_json()
+                time.sleep(1)
+        print(result)
```

Comment:
```suggestion
        if self.is_live:
            while result['provisioningState'] == 'ConfiguringAAD':
                result = self.cmd('az redis show -n {name} -g {rg}').get_output_in_json()
                time.sleep(1)
```
Are these `print(result)`s for debug?

Reply:
Yes I have removed them now